src/Task/Git/CommitMessage.php

```diff
@@ -172,6 +172,10 @@ private function enforceTextWidth(ContextInterface $context)
         }
 
         foreach (array_slice($lines, 2) as $index => $line) {
+            if (strpos($line, '#') === 0) {
```

Comment:
It's also possible that the comment is above the message (e.g. if you rebase)
This might result in a subject line being a comment.

Maybe the comments should always be dropped before doing any of these commit message cehcks instead of only during the max length check?